_opensearch/install/compatibility.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Compatibility
+parent: Install OpenSearch
+nav_order: 2
+---
+
+# Operating system and JVM compatibility
```

Comment:
I think this should be split into two sections, OS and JVM

Reply:
This is already such a short page that I don't see a lot of utility in breaking things up further, especially if you think about it as "all the stuff you have to have in place prior to installing."